docs/source/user_guides/Gateway/Gateway.rst

```diff
@@ -718,12 +724,24 @@ Cloud Sync
 
    filer.sync.unsuspend()
 
-.. automethod:: cterasdk.edge.sync.Sync.refresh
+.. automethod:: cterasdk.edge.sync.Sync.exclude_files
    :noindex:
 
 .. code-block:: python
 
-   filer.sync.refresh()
+   filer.sync.exclude_files(['exe', 'cmd', 'bat'])  # exclude file extensions
+
+   filer.sync.exclude_files(filenames=['Cloud Sync.lnk', 'The quick brown fox.docx'])  # exclude file names
+
+   """Exclude file extensions and file names"""
+   filer.sync.exclude_files(['exe', 'cmd'], ['Cloud Sync.lnk'])
```

Comment:
Consider adding an example for the custom rules

Reply:
Done